docusaurus.config.js

```diff
@@ -431,8 +431,19 @@ const config = {
           label: 'Integrations',
         },
         {
+          type: 'dropdown',
           label: 'CLI',
-          to: '/cli/',
+          position: 'left',
```

Comment:
Instead of setup can you use the term "Reference Docs"

Reply:
Updated to "Reference Docs"